docs/modules/ROOT/pages/openshift-service-mesh/tutorial-bookinfo.adoc

```diff
@@ -0,0 +1,227 @@
+
+The ServiceMeshMemberRoll associates OpenShift projects with ServiceMeshControlPlanes.
+Each project can belong to only one Control Plane, but one Control Plane can manage multiple projects.
+The ServiceMeshControlPlane can only manage services in projects that are associated with it via a ServiceMeshMemberRoll.
+
+. Create a new project to contain the application that shall be part of the service mesh
++
+[source,shell]
+--
+oc new-project demo-ossm-bookinfo
+--
+. Create a ServiceMeshMemberRoll in the control plane namespace to associate this new project with the control plane
++
+[source,shell]
+--
+oc create -n demo-ossm-controlplane -f - <<EOF
+apiVersion: maistra.io/v1
+kind: ServiceMeshMemberRoll
+metadata:
+  name: default
+spec:
+  members:
+    - demo-ossm-bookinfo
+EOF
+--
+
+. Verify the ServiceMeshMemberRoll was successfully created
++
+[source,shell]
+--
+oc get smmr -n demo-ossm-controlplane default
+--
++
+A status of `Configured` indicates that installation was successful.
+
+
+== Step 3: Deploying the bookinfo demo application
+
+`bookinfo` is a small sample application which is built from microservices.
+It displays information about a book.
+The main `productpage` service calls other microservices to collect information and reviews pertaining to a book.
+
+. Deploy the Bookinfo application from its manifest:
++
+[source,shell]
+--
+oc apply -n demo-ossm-bookinfo -f https://raw.githubusercontent.com/Maistra/istio/maistra-2.3/samples/bookinfo/platform/kube/bookinfo.yaml
+--
+
+. Deploy the ingress gateway, which allows public access to the application
++
+[source,bash]
+--
+oc apply -n demo-ossm-bookinfo -f https://raw.githubusercontent.com/Maistra/istio/maistra-2.3/samples/bookinfo/networking/bookinfo-gateway.yaml
+--
+
+. Deploy the destination rules for the application, which configure the routing between the individual microservices
++
+[source,bash]
+--
+oc apply -n demo-ossm-bookinfo -f https://raw.githubusercontent.com/Maistra/istio/maistra-2.3/samples/bookinfo/networking/destination-rule-all.yaml
+--
+
+. Verify all pods are ready
++
+[source,bash]
+--
+oc get pods -n demo-ossm-bookinfo
+--
+
+== Step 4: Enable sidecar injection
+To avoid interference between different operators, the OpenShift Service Mesh controller refuses to touch any deployments that aren't annotated a certain way.
+We need to add this annotation to all our deployments to allow the controller to inject the necessary sidecars.
+
+. Check all the deployments of our application
++
+[source,bash]
+--
+oc get deployment -n demo-ossm-bookinfo
+--
+
+. Add the annotation to all deployments
++
+[source,bash]
+--
+oc get deployment -n demo-ossm-bookinfo -o NAME | while read line ; do oc annotate "$line" -n demo-ossm-bookinfo "sidecar.istio.io/inject=true" ; done
+--
++
+NOTE: The bookinfo application technically already has sidecar injection enabled by default.
+However, the step is highlighted here anyways, since it's necessary on any custom workloads.
```

Comment:
This probably gets rendered correctly, but for multi-line admonitions we usually use the form which allows arbitrary contents:

```suggestion
[NOTE]
====
The bookinfo application technically already has sidecar injection enabled by default.
However, the step is highlighted here anyways, since it's necessary on any custom workloads.
====
```